### PR TITLE
real_estate - RMcNeil

### PR DESCRIPTION
**Pull request by ryan-mcneil**

### CODE REVIEW

File: test/room_test.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  def test_it_exists
+    room = Room.new(:bedroom, 10, 13)
+    assert_instance_of Room, room
+  end
+
+  def test_it_has_attributes
+    room = Room.new(:bedroom, 10, 13)
+    assert_equal :bedroom, room.category
+    assert_equal 10, room.length
+    assert_equal 13, room.width
```
Comment (BrianZanti) [inline test/room_test.rb]: These aren't in the spec, so you don't necessarily need them, but it's not a bad thing

File: test/house_test.rb [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+class HouseTest < Minitest::Test
+
+  def test_it_exists
+    house = House.new("$400000", "123 sugar lane")
+    assert_instance_of House, house
+  end
+
+  def test_it_has_attributes
+    house = House.new("$400000", "123 sugar lane")
+    assert_equal "$400000", house.price
```
Comment (BrianZanti) [inline test/house_test.rb]: According to the spec, this should come back as an integer. I would rather see you write the test correctly and see it fail rather than have a test that passes but is written incorrectly.

File: test/house_test.rb [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    room_2 = Room.new(:bedroom, 11, 15)
+    room_3 = Room.new(:living_room, 25, 15)
+    room_4 = Room.new(:basement, 30, 41)
+    house.add(room_1)
+    house.add(room_2)
+    house.add(room_3)
+    house.add(room_4)
+    assert_equal [room_1, room_2], house.rooms_from_category(:bedroom)
+    assert_equal [room_4], house.rooms_from_category(:basement)
+    assert_equal [], house.rooms_from_category(:kitchen)
```
Comment (BrianZanti) [inline test/house_test.rb]: I like it! This is a great example of edge case testing.
